Tools/Helm/Charts/redis-sentinel/templates/netpol.yml

```diff
@@ -0,0 +1,28 @@
+{{- $name := include "standard.name" . -}}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ $name }}-netpol
+  labels: {{ include "standard.labels" . | nindent 4 }}
+spec:
+  podSelector:
+    matchLabels:
+      name: {{ $name }}
+  ingress:
+    - from:
+        - podSelector:
+            matchLabels:
+              name: {{ $name }}
+      ports:
+        - port: {{ .Values.redisPort }}
+          protocol: TCP
+        - port: {{ .Values.sentinelPort }}
+          protocol: TCP
+    - from:
+        - podSelector:
+            matchLabels: {}
+      ports:
+        - port: {{ .Values.redisPort }}
+          protocol: TCP
+        - port: {{ .Values.sentinelPort }}
+          protocol: TCP
```

Comment:
Is the first rule made redundant by the second?

Reply:
yes, cleaning up